package.json

```diff
@@ -19,9 +19,9 @@
     }
   },
   "files": [
-    "dist",
-    "src",
-    "types"
+    "dist/",
```

Comment:
i recommend updating it to what has been done in `target-decisioning-engine` 👇 
```
    "dist/*",
    "types/*",
    "src/**/!(*.spec).js",
```
see https://github.com/adobe/target-nodejs-sdk/blob/main/packages/target-decisioning-engine/package.json#L8

Reply:
I am running npm pack locally to verify what will be published. 
[The ](https://docs.npmjs.com/cli/pack)[pack](https://docs.npmjs.com/cli/pack)[ command](https://docs.npmjs.com/cli/pack) creates a .tgz file exactly the way it would if you were going to publish the package to npm. It pulls the name and version from package.json, resulting in a file like package-name-0.0.0.tgz. It's exactly the file that would be uploaded to npm if you published it. 

<img width="288" alt="Screenshot 2023-04-27 at 12 02 16 PM" src="https://user-images.githubusercontent.com/14132091/234965584-f0a045f9-1fef-4de6-9628-ce4b4021ae99.png">